client/landing/jetpack-cloud/components/sidebar/heading.tsx

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
instead of creating our own can we reuse the one that is provided by calypso already? 
See `client/layout/sidebar/heading.jsx`